scripts/module_list/module_overview.py

```diff
@@ -0,0 +1,175 @@
+import os
+import subprocess
+from typing import Union, Tuple
+from mdutils.mdutils import MdUtils
+import numpy as np
+
+
+# --------------------------------------------------------------------------------------------------------
+# Module bash API
+# --------------------------------------------------------------------------------------------------------
+
+def module(*args, filter_fn=lambda x: x) -> np.ndarray:
+    """
+    API to call the module command.
+
+    @param args: Extra arguments for the module command.
+    @param filter_fn: Filter function on the ouput.
+    @return:
+    """
+    lmod = os.getenv('LMOD_CMD')
```

Comment:
No need to implement in this PR, but we should add some error handling here.
What if `$LMOD_CMD` is not set (or points to a non-existing path)?
What if the exit code of the Lmod command is non-zero?